app.js

```diff
@@ -0,0 +1,8 @@
+function onoff() {
+  let element = document.getElementById("getid");
+    element.classList.toggle("active");
+    element.removeEventListener('active')
+   

```

Comment:
Additionally, I assume that we need to use eventListener on a body container. It will make a sense when you extend your solution with sub menu.

Reply:
I will read the removeEventListener again